pkg/apply/task/inv_add_task_test.go

```diff
@@ -122,7 +122,7 @@ func TestInvAddTask(t *testing.T) {
 			applyIds, err := object.UnstructuredsToObjMetas(tc.applyObjs)
 			require.NoError(t, err)
 
-			if !object.SetEquals(applyIds, task.Identifiers()) {
+			if !object.IsSetEqual(applyIds, task.Identifiers()) {
```

Comment:
Remove this change. If you want to advocate for a name change, please do it in another PR.